src/system/kernel_linux/Team.cpp

```diff
@@ -30,16 +47,80 @@ _kern_kill_team(team_id team)
 status_t
 _kern_get_team_info(team_id id, team_info *info)
 {
-	info->team = id;
-	return B_OK;
+  	char proc_path[500];
+    char buffer[MAX_LINE_LENGTH];
+    char command_proc_path[500];
+    sprintf(proc_path, "/proc/%d/status", id);
+    sprintf(command_proc_path, "/proc/%d/cmdline", id);
+  	FILE *fileid = fopen(proc_path, "r");
+    int file_command_id = open(command_proc_path, O_RDONLY);
+    if( fileid == NULL ) {
+		return B_ERROR;
+	}
+    int i=0;
+    while( !feof(fileid) ){
+        fgets(buffer, MAX_LINE_LENGTH, fileid);
+        if( i == UID_LINE ) {
+            sscanf(buffer, "Uid:\t%u\t", &info->uid);
+        } else if( i == GID_LINE ) {
+            sscanf(buffer, "Gid:\t%u\t", &info->gid);
+        } else {
+            if(strncmp(buffer, "Threads:", sizeof("Threads:")) == 0){
+                sscanf(buffer, "Threads:\t%d", &info->thread_count);
+                break;
+            }
+        }
+        i++;
+    }
+    int bytes_read = read(file_command_id, buffer, MAX_LINE_LENGTH);
+    int argCount = 0;
+    int j = 0;
+    while( j < bytes_read ){
```

Comment:
Why use a while when you can use a for :)

Reply:
because i remember to have seen everywhere:  "whiile (!feof(fp))" from when i was studying c :D